website/docs/tutorial/getting-set-up.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Getting set up
+id: getting-set-up
+description: "Set up your environment so you can start using dbt."
+---
+
+Learn how to turn the following SQL query into a tested, documented, and deployed dbt project. You can preview [the generated documentation](https://www.getdbt.com/getting-started-tutorial/#!/overview) for the project you're about to build. You'll need a working knowledge of SQL to complete this tutorial.
```

Comment:
I think we want to give an time estimation here. I usually say our getting started takes about 1 hour to do but I don't know if @coapacetic would agree with me.